packages/shared-components/PostList/index.jsx

```diff
@@ -92,7 +92,7 @@ const renderPost = ({
     profileServiceType,
   };
   let PostComponent = postTypeComponentMap.get(post.type);
-  PostComponent = PostComponent || TextPost;
+  PostComponent = PostComponent || Post;
```

Comment:
can this not just be replaced with (all good if you are still in progress and this is part of step5/6 😉)
```suggestion
  PostComponent = Post;
```

Reply:
This one still has Story variation, I'll make it look nicer in a future PR!